docs/dag-serialization.rst

```diff
@@ -58,11 +58,16 @@ Add the following settings in ``airflow.cfg``:
     [core]
     store_serialized_dags = True
     min_serialized_dag_update_interval = 30
+    min_serialized_dag_fetch_interval = 10
+    store_dag_code = True
 
 *   ``store_serialized_dags``: This flag decides whether to serialise DAGs and persist them in DB.
     If set to True, Webserver reads from DB instead of parsing DAG files
 *   ``min_serialized_dag_update_interval``: This flag sets the minimum interval (in seconds) after which
     the serialized DAG in DB should be updated. This helps in reducing database write rate.
+*   ``min_serialized_dag_fetch_interval``: This flag sets the minimum interval (in seconds) after which
+    the serialized DAG will be fetched from DB from the Webserver. This helps in reducing database read rate.
+    This config controls when your DAGs are updated in the Webserver.
```

Comment:
This isn't quite clear -- this settings controls how often it will be re-fetched from the DB when it's _already loaded in the dagbag_, but that part isn't clear in the docs yet. We should probably say something like "Setting this higher will reduce load on the DB, but at the expense of displaying a possibly stale cached version"

Reply:
Updated in https://github.com/apache/airflow/pull/9851/commits/02856d63a5d500788c8aab7c49a235b036800364